### ISSUE TITLE
"BowFly" option for Flight

### ISSUE THREAD

**Issue by VitorISs** [ImpactDevelopment/ImpactClient#270]
A module that shoots arrows on yourself then you can fly with the knockback. (Does bypass NCP) https://i.gyazo.com/320c6973fcc5dc9a623fd79837114f2e.mp4

**Comment by VitorISs** [comment]
i accidentally pressed enter and send the issue without describing it e.e
but now its all right

**Comment by zPeanut** [comment]
and how the fuck do we know how it works, you cant just suggest shit that is in other client and suggest it into here just because it bypasses ncp aac whatever, thats bullcrap and you should stop doing it. other suggestions you left were like "<name> add this to impact, bypasses lastest aac" etc etc etc, 1st off brady or leaf doesnt know how the fuck that works and second of all you just fill the suggestions section with useless crap that nobody can start with. you are blocked from this issue report and all your suggestions that are repetetive are being closed and marked as invalid.

**Comment by VitorISs** [comment]
hmmmm..... 
"and how the fuck do we know how it works" really ? lol
it ~shoots arrows on urself~ woooooooooooooooow
i made this issue because brady said NCP is the priority now... and a PRETTY SIMPLE flight module that BYPASS NCP looks a good suggestion..
and... repetetive ? WHICH issue is repetitive? wtf are you saying l o l
all of my suggestions looks pretty normal.. and are marked as "Enhancement"...
ppl use this ISSUE SYSTEM to report bugs or make Suggestions... S U G G E S T I O N S.
i said THERE'S A WAY to bypass aac... IF he want to make some bypass, or whatever. i actually don't want it.. i couldn't care less about aac.

but.. if you want me stop to making issues about improvements/useful modules that could be added, ok, i'll stop..

**Comment by VitorISs** [comment]
i don't think you can make a 100% functional flight... but at least you can make better longjumps and stuff like that... works pretty well
